diff --git a/source/app.d b/source/app.d
index f93da61..00e1e42 100644
--- a/source/app.d
+++ b/source/app.d
@@ -106,7 +106,7 @@ shared static this()
 	// start the web server
  	auto settings = new HTTPServerSettings;
 	settings.hostName = hostname;
-	settings.bindAddresses = ["127.0.0.1"];
+	settings.bindAddresses = ["0.0.0.0"];
 	settings.port = 8005;
 	settings.sessionStore = new MemorySessionStore;
 
